common/thorhelper/CMakeLists.txt

```diff
@@ -110,6 +110,10 @@ include_directories (
          ${HPCC_SOURCE_DIR}/testing/unittests
          ${HPCC_SOURCE_DIR}/system/security/shared
          ${HPCC_SOURCE_DIR}/fs/dafsclient
+#ifdef _USE_PARQUET
```

Comment:
The #ifdef is not correct - since this is a cmake file (it will be treated as a comment).

You need something like:
```
if (USE_PARQUET)
  include_directories ( .... )
endif()
```
See common/dllserver/CMakeList.txt for an example.
Same also goes for line 131 etc.